static/styles/portico.scss

```diff
@@ -743,7 +743,7 @@ input.text-error {
 }
 
 .portico-header .dropdown ul li:hover {
-    background: linear-gradient(to bottom, #08c, #0077b3);
+    background: linear-gradient(to bottom, #08c, hsl(200, 100%, 35%));
```

Comment:
You mentioned HSL is incompatible with gradients so marking this.

Reply:
They should be fine for linear gradients, but not filters. Fixed the `#08c` not being HSL 👍